thermalnetwork/tests/test_network.py

```diff
@@ -141,6 +141,41 @@ def test_network_ghe_proportional(self):
             output_path,
         )
 
+        # -- Check
+        expected_depth_of_boreholes = pytest.approx(133, 2)
+        for ghe_id in output_path.iterdir():
+            if ghe_id.is_dir():
+                sim_summary = json.loads((ghe_id / "SimulationSummary.json").read_text())
+
+                assert sim_summary["ghe_system"]["active_borehole_length"]["value"] == expected_depth_of_boreholes
+
+        # -- Clean up
+        # Restore the original borehole length and number of boreholes.
+        sys_param_ghe = json.loads(self.system_parameter_path_13_buildings_proportional_ghe.read_text())
+        proportional_ghe_specific_params = sys_param_ghe["district_system"]["fifth_generation"]["ghe_parameters"][
```

Comment:
Also curious whether `proportional_ghe_specific_params` is accurate, or just a copy/paste from above?

Reply:
You are right that the work `proportional` has no meaning here so I removed it from the variable name.  But the code is correct.